pop-api/integration-tests/contracts/create_token_in_constructor/lib.rs

```diff
@@ -14,22 +14,23 @@ mod create_token_in_constructor {
 
 	#[ink(storage)]
 	pub struct Fungible {
-		id: TokenId,
+		token: TokenId,
 	}
 
 	impl Fungible {
 		#[ink(constructor, payable)]
-		pub fn new(id: TokenId, min_balance: Balance) -> Result<Self> {
-			let contract = Self { id };
+		pub fn new(token: TokenId, min_balance: Balance) -> Result<Self> {
+			let contract = Self { token };
```

Comment:
This should stay `id` as the token is not created yet. Same for in the contract struct